docs/examples/using-terraform-plugin.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: default
+title: Spawn a virtual machine on each Cloud using Terraform!
+nav_order: 10
+parent: Examples
+---
+
+# Introduction
+This example will go through how to spawn a virtual machine using Terraform with Direktiv. We will use a publicly accessible [git](https://github.com/vorteil/terraform-examples). Then proceed to message a Discord webhook to let everyone know the instance is alive.
```

Comment:
This example will detail how to user Direktiv with Terraform to create a virtual machine. To do this, we will use examples listed on the public [git repository](https://github.com/vorteil/terraform-examples). After creating the virtual machine, a message will be sent to a Discord webhook, resulting in the message being posted to a Discord server text channel.